app/src/main/res/layout/add_meeting_form.xml

```diff
@@ -15,7 +15,7 @@
         android:layout_width="408dp"
         android:layout_height="32dp"
         android:gravity="center"
-        android:text="New Meeting"
+        android:text="@string/newMeeting"
```

Comment:
Good idea.  Putting Strings in strings.xml is more maintainable, and better for internationalization.